rust/remotivelabs-broker/README.md

```diff
@@ -6,7 +6,9 @@ gRPC bindings are generated from [this source](../../proto/) when compiling the
 
 This project is contributed by _Niclas Lind_ (@niclaslind).
 
-## Publish / Subscribe example
+## Examples
+
+### Publish / Subscribe example
 See `examples/pub_client.rs` and `examples/sub_client.rs` for a sample on how to use the library.
 
 The samples work together and will connect to a _RemotiveBroker_ running on `localhost`. First start the subscription sample:
```

Comment:
mention that this example is using the "virtual" interface over the RemotiveBroker

Reply:
This is updated in #37